retrospective-update/master.py

```
@@ -0,0 +1,110 @@
+
+        CL.log_message('RUNNING', 'updating monthly zarrs')
+        f.update_monthly_zarrs(LOCAL_DAILY_ZARR, S3_MONTHLY_TIMESTEPS, S3_MONTHLY_TIMESERIES)
+        
+        CL.log_message('RUNNING', 'updating annual zarrs')
+        f.update_yearly_zarrs(LOCAL_HOURLY_ZARR, S3_ANNUAL_TIMESTEPS, S3_ANNUAL_TIMESERIES, S3_ANNUAL_MAXIMUMS)
+
+        CL.log_message('RUNNING', 'cleaning up current run')
+        f.cleanup(volume_directory, RUNOFF_DIR, INFLOWS_DIR, OUTPUTS_DIR)
+
+        CL.log_message('COMPLETE')
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)